EiDiA_frontend/src/components/exportView/subcomponents/SetValueSection.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import TextField from '@material-ui/core/TextField';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import {MdAdd} from "react-icons/all";
+
+const styles = {
+    textField: {
+        left: "5px",
+    },
+    button: {
+        margin: "15px",
+        right: 0
+    }
+}
+
+function createData(variable, value, source) {
+    return {variable, value, source};
+}
+
+function convertData(variables, value = null) {
+    let out = [];
+    for (let k of Object.keys(variables)) {
+        let variable = variables[k];
+        out.push(createData(k, variable.value, variable.source));
+    }
+    return out;
+}
+
+export default class SetValueSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            input: "",
+        }
+
+        this.handleChangeTextfield = this.handleChangeTextfield.bind(this);
+        this.setValueToVariable = this.setValueToVariable.bind(this);
+    }
+
+    setValueToVariable() {
+        this.props.onAction3(this.state.input);
+    }
+
+
+    handleChangeTextfield(event) {
+        this.setState({input: event.target.value});
+    }
+
+    render() {
+        const variableState = this.props.variables;
+        const variableKeys = Object.keys(variableState);
+        const rows = convertData(variableState);
+        return (
+            <div>
+                <FormControl style={{margin: "5px"}}>
```

Comment:
```suggestion
                <FormControl style={{margin: "5px", width: '100%', padding: '0 10px 0 0'}}>
```
fixes the dropdown's width